src/components/ticker/SignalGroupCard.tsx

```diff
@@ -0,0 +1,89 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faSignalMessenger } from '@fortawesome/free-brands-svg-icons'
+import { faGear, faPause, faPlay, faTrash } from '@fortawesome/free-solid-svg-icons'
+import SignalGroupModalForm from './SignalGroupModalForm'
+
+interface Props {
+  ticker: Ticker
+}
+
+const SignalGroupCard: FC<Props> = ({ ticker }) => {
+  const { token } = useAuth()
+  const { deleteTickerSignalGroup, putTickerSignalGroup } = useTickerApi(token)
+  const [open, setOpen] = useState<boolean>(false)
+
+  const queryClient = useQueryClient()
+
+  const signalGroup = ticker.signalGroup
+
+  const handleDisconnect = useCallback(() => {
+    deleteTickerSignalGroup(ticker).finally(() => {
+      queryClient.invalidateQueries({ queryKey: ['ticker', ticker.id] })
+    })
+  }, [deleteTickerSignalGroup, queryClient, ticker])
+
+  const handleToggle = useCallback(() => {
+    putTickerSignalGroup({ active: !signalGroup.active }, ticker).finally(() => {
+      queryClient.invalidateQueries({ queryKey: ['ticker', ticker.id] })
+    })
+  }, [signalGroup.active, putTickerSignalGroup, queryClient, ticker])
+
+  const groupLink = (
+    <Link href={signalGroup.groupInviteLink} rel="noreferrer" target="_blank">
+      {signalGroup.groupName}
+    </Link>
+  )
+
+  return (
+    <Card>
+      <CardContent>
+        <Stack alignItems="center" direction="row" justifyContent="space-between">
+          <Typography component="h5" variant="h5">
+            <FontAwesomeIcon icon={faSignalMessenger} /> Signal Group
+          </Typography>
+          <Button onClick={() => setOpen(true)} size="small" startIcon={<FontAwesomeIcon icon={faGear} />}>
+            Configure
+          </Button>
+        </Stack>
+      </CardContent>
+      <Divider variant="middle" />
+      <CardContent>
+        {signalGroup.connected ? (
+          <Box>
+            <Typography variant="body2">You are connected with the Signal group.</Typography>
+            <Typography variant="body2">Your Signal group invite link: {groupLink}</Typography>
+          </Box>
+        ) : (
+          <Box>
+            <Typography variant="body2">You are not connected with the Signal group.</Typography>
+            <Typography variant="body2">New messages will not be published to the group and old messages can not be deleted anymore.</Typography>
+          </Box>
+        )}
+      </CardContent>
+      {signalGroup.connected ? (
+        <CardActions>
+          {signalGroup.active ? (
+            <Button onClick={handleToggle} size="small" startIcon={<FontAwesomeIcon icon={faPause} />}>
+              Disable
+            </Button>
+          ) : (
+            <Button onClick={handleToggle} size="small" startIcon={<FontAwesomeIcon icon={faPlay} />}>
+              Enable
+            </Button>
+          )}
+          <Button onClick={handleDisconnect} size="small" startIcon={<FontAwesomeIcon icon={faTrash} />}>
+            Disconnect
```

Comment:
Disconnecting the bridge causes the group data to be lost. Not sure whether we want to allow this at all.